### PR TITLE
Adding method to return a list of MailTemplate objects

### PR DESCRIPTION
**Pull request by wellyfrs**
Adding method to return a list of MailTemplate objects (for `rainlab/translate-plugin` list translatable messages in the backend). See: https://github.com/rainlab/translate-plugin/issues/87

### CODE REVIEW

**Comment by LukeTowers** [comment]
This pull request has been made to the wrong branch. Please review [the contributing guidelines](https://github.com/octobercms/october/blob/master/CONTRIBUTING.md#pull-requests) as all PRs need to be made to the develop branch.

I'll fix it for you this time, but please ensure you make any future PRs to the develop branch, not the master branch.

File: modules/system/models/MailTemplate.php [hunk 1]
```diff
@@ -60,6 +60,23 @@ public static function listAllTemplates()
...
+     * Returns a list of all mail templates.
+     * @return array Returns an array of the MailTemplate objects.
+     */
+    public static function allTemplates()
+    {
+        $result = [];
+        $codes = array_keys(self::listAllTemplates());
+
+        foreach ($codes as $code) {
+
```
Comment (LukeTowers) [inline modules/system/models/MailTemplate.php]: What's with the extra whitespace here?

Comment (wellyfrs) [reply modules/system/models/MailTemplate.php]: Line 72 can be removed. Thanks for the branch move!

Comment (LukeTowers) [reply modules/system/models/MailTemplate.php]: It's easier for me if you do that yourself on your branch so that I can merge it in via the PR.

Comment (wellyfrs) [reply modules/system/models/MailTemplate.php]: Done!

**Pull request merged**